Firestore/core/src/model/value_util.h

```diff
@@ -121,15 +121,15 @@ bool IsMinValue(const google_firestore_v1_Value& value);
  * Firestore comparison logic always return true for `MaxValue() > v`, for any
  * v users can create, regardless `v`'s type and value.
  */
-nanopb::Message<google_firestore_v1_Value> MaxValue();
+google_firestore_v1_Value MaxValue();
```

Comment:
You should update the comments in this file to clarify who owns the memory.

Reply:
Done.